test/e2e/system_connection_test.go

```diff
@@ -281,6 +281,17 @@ var _ = Describe("podman system connection", func() {
 			_, err = Start(cmd, GinkgoWriter, GinkgoWriter)
 			Expect(err).ToNot(HaveOccurred())
 
+			// export the container_host env var and try again
+			err = os.Setenv("CONTAINER_HOST", fmt.Sprintf("ssh://%s@localhost", u.Username))
+			Expect(err).ToNot(HaveOccurred())
+
+			cmd = exec.Command(podmanTest.RemotePodmanBinary, "ps")
+			_, err = Start(cmd, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = os.Unsetenv("CONTAINER_HOST")
```

Comment:
this needs to be defer os.Unsetenv("CONTAINER_HOST") right after the setenv call, otherwise you will leak the env on test errors.